ui/src/app/views/workspace/component.spec.ts

```diff
@@ -62,10 +70,17 @@ describe('WorkspaceComponent', () => {
   beforeEach(fakeAsync(() => {
     TestBed.configureTestingModule({
       imports: [
-        RouterTestingModule
+        RouterTestingModule,
+        ClarityModule.forRoot()
       ],
       declarations: [
-        WorkspaceComponent
+        WorkspaceComponent,
+        // Datagrid,
```

Comment:
Please leave these out for now (I'm assuming they're something you're expecting to use later?). Or, include a clear TODO like "TODO(brew) Enable these to support the sortable table in WorkspaceComponent." (My goal is that at any commit, only live code & accurate docs exist in master.)